File: backend-model/src/main/java/outcobra/server/model/dto/MarkDto.java
package outcobra.server.model.dto;

import outcobra.server.model.Mark;
import outcobra.server.model.interfaces.MappableDto;
import outcobra.server.model.interfaces.Mapper;

public class MarkDto implements MappableDto<MarkDto, Mark> {
    @Override
    public Mapper<MarkDto, Mark> getMapper() {
        throw new UnsupportedOperationException("not implemented");
    }

    @Override
    public Mark toEntity() {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: backend/src/main/kotlin/outcobra/server/model/dto/emptyDto/MarkDto.kt
package outcobra.server.model.dto.emptyDto

import outcobra.server.model.Mark
import outcobra.server.model.mapper.MappableDto
import outcobra.server.model.mapper.Mapper

class MarkDto : MappableDto<MarkDto, Mark> {
    override fun getMapper(): Mapper<MarkDto, Mark> {
        throw UnsupportedOperationException("not implemented")
    }

    override fun toEntity(): Mark {
        throw UnsupportedOperationException("not implemented")
    }
//TODO Implement
}